spec/factories/authentication.rb

```diff
@@ -1,6 +1,5 @@
 FactoryGirl.define do
-  factory :authentication do
-    type        "AuthUseridPassword"
+  factory :authentication, :class => "AuthUseridPassword" do
```

Comment:
This doesn't need `:class`, it should use the default `:class` of `Authentication`.